cypress/integration/settings/open_close_calendar_spec.js

```diff
@@ -36,7 +47,7 @@ describe('Filter calendar periods', function() {
 describe('Actions on selected item', function() {
   it('open/close action - close periods', function() {
     cy.executeHeaderActionWithDialog('C_Period_Process');
-    cy.selectInListField('PeriodAction', 'Close Period', true /*modal*/, '/rest/api/process/');
+    cy.selectInListField('PeriodAction', closePeriod, true, '/rest/api/process/');
```

Comment:
I think here we can remove `'/rest/api/process/'` and it should still work since `cy.selectInListField` expects by default a patch from `RewriteURL.Generic`.

Reply:
done.